src/components/Proposals/Description.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { GU, IdentityBadge, useTheme } from '@aragon/ui'
+import { propTypes } from '@aragon/ui/dist/index-46d0e707'
```

Comment:
Should we have  our own proptypes here or export them properly on aragonUI?

Reply:
Ohhh nice spot! my editor auto imported this from the wrong place 🤦  thanks!

Reply:
Fixed in https://github.com/aragon/network-dashboard/pull/74/commits/f6b10c9365a20c11470edb5595df9e8d67f52e4f